dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/RateLimiter.java

```diff
@@ -0,0 +1,30 @@
+package com.salesforce.dockerfileimageupdate.utils;
+
+import io.github.bucket4j.Bandwidth;
+import io.github.bucket4j.Bucket;
+import io.github.bucket4j.Bucket4j;
+import io.github.bucket4j.Refill;
+import java.time.Duration;
+
+public class RateLimiter {
```

Comment:
We should have javadocs that describes what this class is an what its used for?

Reply:
Yes, the ide will generate post of this if you start your comment in the class with `/** `